settings.gradle.kts

```diff
@@ -26,6 +26,15 @@ dependencyResolutionManagement {
     }
 }
 
+// Set up aws-crt-kotlin as a composite build
+val localAwsCrtKotlin = file("../aws-crt-kotlin")
+if (localAwsCrtKotlin.exists()) {
+    println("Including aws-crt-kotlin as a composite build")
+    includeBuild(localAwsCrtKotlin)
+} else {
+    println("Could not find aws-crt-kotlin in a sibling directory, not including it")
+}
+
```

Comment:
**Question**: This sets up a composite build with **aws-crt-kotlin** for **smithy-kotlin** but what if I'm building **aws-sdk-kotlin** and I'm already setting `compositeProjects=../smithy-kotlin,../aws-crt-kotlin`? Does the CRT project get built twice? Does it fail?

Reply:
I don't configure `compositeProjects` at all and my **aws-sdk-kotlin** build gets resolved correctly with all three projects included